versions/3.0.md

```diff
@@ -778,6 +780,41 @@ Field Name | Type | Description
 <a name="parameterEnum"></a>enum | [*] | See http://json-schema.org/latest/json-schema-validation.html#anchor76.
 <a name="parameterMultipleOf"></a>multipleOf | `number` | See http://json-schema.org/latest/json-schema-validation.html#anchor14.
 
+In order to support alternate ways of serializing parameters, a set of `style` values are defined.
+
+`style` | [`type`](#dataTypeType) |  `in` | Comments
+----------- | ------ | -------- | --------
+matrix |  `string`, `array`, `object` |  `path` | Path-style parameters defined by [RFC6570](https://tools.ietf.org/html/rfc6570#section-3.2.7) 
```

Comment:
Everywhere it says `string` as the `type`, it should be `primitive` (it can be any primitive)